Source/Fuse.Controls.CameraView/Android/Camera.uno

```diff
@@ -155,6 +155,32 @@ namespace Fuse.Controls.Android
 			return new AndroidPhotoOptionPromise(this).Visit(options);
 		}
 
+		class AndroidCameraFocusPointPromise : Promise<CameraFocusPoint> {
+			Camera _camera;
+			string _parameters;
+
+			public AndroidCameraFocusPointPromise(Camera camera) {
+				_camera = camera;
+				_parameters = _camera.SaveParameters();
+			}
+				
+			public Future<CameraFocusPoint> doFocus(double x, double y, int cameraWidth, int cameraHeight, int isFocusLocked) {
+				try {
+					_camera.SetCameraFocusPointNow(x,y, cameraWidth, cameraHeight, isFocusLocked);
+					
+					Resolve((CameraFocusPoint)1);
```

Comment:
This seems to cast 1 to a non-existent enum-value, which doesn't make a whole lot of sense to me. How about adding a new corresponding enum value and using that instead? Then some semantics would be attached to this, instead of a magical "1"...

Reply:
Actually, shouldn't this just be `Resolve(CameraFocusPoint.OK)`? This looks like a success-path...

Reply:
Yeah, I think initially we were gonna return an x and y point but it wasn't necessary, so I guess we can remove the enums?

Reply:
Sounds good